stories/annotations/boxPlot.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import { Meta } from '@storybook/react';
+
+import {
+  AnnotationBoxPlotProps,
+  BoxPlot,
+} from '../../src/components/Annotations/BoxPlot';
```

Comment:
This should be imported from `'../../src'`. Need to export in https://github.com/zakodium-oss/react-plot/blob/main/src/components/Annotations/Annotation.tsx

Reply:
to be imported from ../../src we must use annotation.BoxPlot

Reply:
fixed in all annotations stories

Reply:
Cool, thanks.
I think it's better to use the public API in the storybook so we don't forget to export and it's easier to copy/paste an example.